src/main/go/chaincode/myCC/myCC.go

```diff
@@ -85,12 +74,80 @@ func (t *SimpleChaincode) Invoke(stub shim.ChaincodeStubInterface) pb.Response{
 		return t.add (stub, args)	
 	}
 	
+	if args[0] == "addMember" {
+		return t.addMember(stub, args)
+	}
+	
+	if args[0] == "queryVerified" {
+		return t.queryVerified(stub, args)
+	}
+	
+	if args[0] == "addConnection" {
```

Comment:
Wondering if we want a seperate chaincode for creating/adding/modifying connections.